experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +97,223 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
+function saturation_with_index(I::T,J::T) where T <: Union{ MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal}
+  R = base_ring(I)
+  R == base_ring(J) || error("Ideals do no live in the same ring.")
+
+  I_sat = saturated_ideal(I)
+  J_sat = saturated_ideal(J)
+
+  I_result,k = Oscar.saturation_with_index(I_sat,J_sat)  ## why is saturation_with_index not exported from Rings/mpoly-ideal.jl?
+  return (ideal(R,gens(I_result)),k)
+end
+
+## THIS SHOULD GO TO mpolyquo-localization.jl
+## it is the localized and/or quotient wrapper to the MPolyIdeal-method
+@attr Bool function is_one(I::Union{MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal})
+  return is_one(saturated_ideal(I))
+end
+
+## WHERE SHOULD THIS GO?
+@Markdown.doc """
+  iterated_quotients(I::T, J::T, b::Int) where T <: MPolyAnyIdeal
+
+Return ``I:J^b`` and ``b``, for a given natural number ``b``.
+Return this for the highest ``b`` such that ``J(I:J^b)== (I:J^(b-1))``, if given ``b`` is zero.
```

Comment:
Would two separate functions make sense? 
For instance 
```
  iterated_quotients(I::T, J::T, b::Int) where T <: MPolyAnyIdeal
Return ``I:J^b`` and ``b``, for a given natural number ``b``.
```

```
iterated_quotients(I::T, J::T) where T <: MPolyAnyIdeal
Return this for the highest ``b`` such that ``J(I:J^b)== (I:J^(b-1))``, if given ``b`` is zero.
```

Reply:
I added a default value and documented as suggested, but did not make it two differen functions as the method is identical up to a small detail.